docs/guides/security.md

```diff
@@ -0,0 +1,53 @@
+# Security and Privacy
+
+One of our main pillars is the Security and Privacy of our customers information. This section highlights some of the specific security measures that we have taken to achieve this objective.
+
+## Document encryption and storage
+
+Protecting the confidentiality, integrity, and availability of our customers' data is an important priority for us. To this end, we employ client-side encryption as a critical measure to protect sensitive data from unauthorized access and breaches.
+
+Client-side encryption is a method of encrypting data before it is transmitted over a network or stored in the cloud. In this approach, the encryption and decryption of the data are performed on the client-side, rather than on the server-side, ensuring that only authorized users with the decryption key can access the data.
+
+By utilizing client-side encryption, we ensure that all documents uploaded by our customers are encrypted before leaving their device. This reduces the risk of data breaches and unauthorized access to sensitive information. Furthermore, since the encryption keys are generated and stored on the client-side, the organization has no access to the contents of the encrypted documents, providing an extra layer of data privacy and security.
+
+In addition to encryption, we have also chosen to store our documents on Swiss servers that are certified by ISO 27001. This certification is a widely recognized standard for information security management, and it provides us with independent verification that our servers meet strict security requirements. These requirements include physical security measures, access controls, network security, and other measures that help to safeguard our data against threats.
+
+> Although Azure is our cloud provider, customers are free to use their own document storage.
```

Comment:
Maybe: customer are free to deploy their own document store instance.

Reply:
Change made.